dozer-sql/src/pipeline/errors.rs

```diff
@@ -84,7 +82,8 @@ pub enum PipelineError {
     IllegalFieldIdentifier(String),
     #[error("Unable to cast {0} to {1}")]
     UnableToCast(String, String),
-
+    #[error("Each SQL Select Statement must have a corresponding INTO clause")]
```

Comment:
Let's make the error message a bit more in the style of the others. So, instead of stating the desired outcome, state what went wrong, e.g. `missing INTO clause for top-level SELECT statement`